contracts/modules/governance/NXMaster.sol

```diff
@@ -38,7 +38,10 @@ contract NXMaster is INXMMaster, Governed {
   bool internal reentrancyLock;
   bool public masterInitialized;
   address public owner;
-  uint public pauseTime;
+  uint public _unused1;
+
+  address public emergencyAdmin;
+  bool public paused;
```

Comment:
I think we can replace `_unused1` with `emergencyAdmin` + `paused` :grin: 
Currently `pauseTime` value is `2419200` which is binary `0b1001001110101000000000`. The first item in a storage slot is stored lower-order aligned (as per solidity docs). I think we can have them in any order and `paused` will be `false` (eight zero bits) and `emergencyAdmin` will be set to a "random" address (whatever we get after conversion) which we can initialize afterwards.

I'm not strongly for this, just an idea. We can fork test it or we can drop it if you consider this as being too dangerous.